docs/content/release_notes/whats_new.md

```diff
@@ -4,10 +4,33 @@ IBM® block storage CSI driver 1.11.0 introduces the enhancements that are detai
 
 **General availability date:** 15 January 2023
 
+
 ## New support for policy-based replication
 
 This version adds support for policy-based replication that was introduced in IBM Spectrum Virtualize 8.5.2 release. Policy-based replication provides a simplified configuration and management of asynchronous replication between two system. To see if your specific product is supported and for more information, see **What's new** > **Getting started with policy-based replication** within your Spectrum Virtualize product documentation on [IBM Documentation](https://www.ibm.com/docs).
 
+## Enhanced dynamic host definition
+
+The IBM® block storage CSI driver 1.11.0 host definition feature now supports the following:
+
+- **CSI Topology feature**
+
+    For more information, see [Configuring for CSI Topology](../configuration/confiugring_toplogy.md).
+
+- **Dynamically configuring host ports**
+
+     Changes in host port hierarchy are now identified automatically updated. For more information, see [Configuring the host definer](../configuration/configuring_hostdefiner.md).
+
+- **I/O groups**
```

Comment:
```suggestion
- **I/O group**
```

Reply:
Why did you make this singular? I'm making sure that there is a reason.
English-wise this is correct because we're writing "this supports i/o groups" but just in a list form.

Reply:
because it's like it in the storage:
![image](https://user-images.githubusercontent.com/45543087/210313302-8eef89ed-76b8-4a1d-ab0a-ecb6e0fa09b0.png)

Reply:
I made some changes for this in in the documentation (what's new and using_hostdefinition_labels.md).
It's important to note that there is a difference between how it is on the storage and how it is talked about in general.